dokka-runners/dokka-gradle-plugin/src/main/kotlin/DokkaExtension.kt

```diff
@@ -83,32 +104,9 @@ constructor(
         )
 
     /**
-     * Dokka Source Sets describe the source code that should be included in a Dokka Publication.
+     * Dokka Source Sets describe the source code that should be included in the generated output.
```

Comment:
Dokka Source Set not only represents the source code, but also its Dokka configuration. I would say, for end-users configuration part is more important in DSL

Reply:
Please could you make a suggestion for what info could be added? A rough version is fine, I can tidy it up.

Reply:
Thanks, I've added more details (and also on `dokkaPublications`)